log-collector/src/test/scala/io/phdata/pulse/logcollector/PulseKafkaConsumerTest.scala

```diff
@@ -73,13 +67,14 @@ class PulseKafkaConsumerTest
   def generateLogMessage(level: String, id: Integer, application: String): String = {
     val random = scala.util.Random
 
-    Map("id" -> Some(s"$id").get,
```

Comment:
Removed the 'id' here because it was causing the second batch of docs to be added in the test "Send two message batches to Solr Cloud", because it would generate similar ids and so documents were updated not added